src/options.c

```diff
@@ -499,14 +500,20 @@ bool get_cfg(options_t *opt, int argc, char *const *argv, bool shadow_enable,
 				opt->vsync = true;
 			}
 			break;
+		P_CASEBOOL(271, crop_shadow_to_monitor);
+		case 272:
+			opt->crop_shadow_to_monitor = true;
+			log_warn("--xinerama-shadow-crop is deprecated. Use "
+				 "--crop-shadow-to-monitor instead.");
```

Comment:
I'd like `--xinerama-shadow-crop has been deprecated, please use --crop-shadow-to-monitor instead` more, but these messages are hardly consistent. :disappointed: